sdk/metrics/+opentelemetry/+sdk/+metrics/View.m

```diff
@@ -21,28 +21,48 @@
     end
 
     methods
-        function obj = View(name, description, unit, instrumentName, instrumentType, ...
-                meterName, meterVersion, meterSchemaURL, attributeKeys, ...
-                aggregation, histogramBinEdges, varargin)
+        function obj = View(options)
+            arguments
+                options.name=""
+                options.description=""
+                options.unit=""
+                options.instrumentName=""
+                options.instrumentType=""
+                options.meterName=""
+                options.meterVersion=""
+                options.meterSchemaURL=""
+                options.attributeKeys=""
```

Comment:
Another option I just tested that works is setting the default value like this: `options.AttributeKeys="UseDefaultAttributes"`

That way if the customer passes in "", it will still filter attributes, but if they don't specify any attributes, it will use the default processor. Let me know which you prefer.

Reply:
I wouldn't use any specific nonempty string, like "UseDefaultAttributes" you suggested. With even the most unlikely string, when there are many users, you may still run into someone actually trying to use that string as an attribute name.